docs/tutorial-01.md

```diff
@@ -0,0 +1,210 @@
+# buildah Tutorial 1
+## Building OCI container images
+
+The purpose of this tutorial is to deomonstrate how buildah can be used to build container images compliant with the [Open Container Initiative](https://www.opencontainers.org/) (OCI) [image specification](https://github.com/opencontainers/image-spec). Images can be built from existing images, from scratch, and using Dockerfiles. OCI images built using the buildah command line tool (CLI) and the underlying OCI based technologies (e.g. [containers/image](https://github.com/containers/image) and [containers/storage](https://github.com/containers/storage)) are portable and can therefore run in a docker environment.
```

Comment:
typo: demonstrate

Reply:
ack done (will recommit when all comments are acked and I figure out /remember how to do it ;-)